code/drasil-docLang/lib/Drasil/Sections/TraceabilityMandGs.hs

```diff
@@ -87,10 +87,11 @@ traceMatRefinement = TraceConfig (mkUid "TraceMatRefvsRef") [plural Doc.dataDefn
 
 -- | Records other requirements. Converts the 'SystemInformation' into a 'TraceConfig'.
 traceMatOtherReq :: SystemInformation -> TraceConfig
-traceMatOtherReq si = TraceConfig (mkUid "TraceMatAllvsR") [x plural +:+ S "on the" +:+
-  plural Doc.dataDefn, plural Doc.thModel, plural Doc.genDefn, plural Doc.inModel]
-  (x titleize' +:+ S "and Other" +:+ titleize' item) [tvDataDefns, tvTheoryModels,
-  tvGenDefns, tvInsModels, tvReqs] [tvGoals, tvReqs] where
+traceMatOtherReq si = TraceConfig (mkUid "TraceMatAllvsR") [plural requirement +:+ 
+  S "and" +:+ pluralNP (goalStmt `onThePP` Doc.dataDefn), plural Doc.thModel, 
+  plural Doc.genDefn, plural Doc.inModel] (x titleize' +:+ S "and Other" +:+ 
```

Comment:
It looks like this assumes that an example will have more than one instance of each model (e.g., instance model). How can we build this list from what is _actually_ present in the document, including (ideally) using the singular form of a model if there is only one instance of it? 

The number of DDs and IMs is present in `SystemInformation`:
https://github.com/JacquesCarette/Drasil/blob/017a0768ba6f6bd2a9e018e3c18cba9b2c47492d/code/drasil-example/projectile/lib/Drasil/Projectile/Body.hs#L103-L123

and it looks like it _might_ be populated the rest of the way with `fillcdbSRS`, but I don't know enough about it to verify this:
https://github.com/JacquesCarette/Drasil/blob/017a0768ba6f6bd2a9e018e3c18cba9b2c47492d/code/drasil-docLang/lib/Drasil/DocumentLanguage.hs#L80-L81